docs/_sidebar.md

```diff
@@ -1,33 +1,33 @@
 - Core Concepts
-    - [Address](core-concepts/Address.md)
-    - [Block](core-concepts/Block.md)
+    - [Address](core-concepts/address.md)
+    - [Block](core-concepts/block.md)
     - [crypto](core-concepts/crypto.md)
     - [encoding](core-concepts/encoding.md)
     - [hierarchical](core-concepts/hierarchical.md)
-    - [Mnemonic](core-concepts/Mnemonic.md)
-    - [Networks](core-concepts/Networks.md)
-    - [PrivateKey](core-concepts/PrivateKey.md)
-    - [PublicKey](core-concepts/PublicKey.md)
-    - [Script](core-concepts/Script.md)
-    - [Transaction](core-concepts/Transaction.md)
-    - [Unit](core-concepts/Unit.md)
-    - [UnspentOutput](core-concepts/UnspentOutput.md)
-    - [Uri](core-concepts/Uri.md)
+    - [Mnemonic](core-concepts/mnemonic.md)
+    - [Networks](core-concepts/networks.md)
+    - [PrivateKey](core-concepts/privatekey.md)
+    - [PublicKey](core-concepts/publickey.md)
+    - [Script](core-concepts/script.md)
+    - [Transaction](core-concepts/transaction.md)
+    - [Unit](core-concepts/unit.md)
+    - [UnspentOutput](core-concepts/unspentoutput.md)
+    - [Uri](core-concepts/uri.md)
```

Comment:
The governance object core concept shown in the main README isn't include here for some reason
```suggestion
    - [Uri](core-concepts/uri.md)
    - [Governance Object](core-concepts/govobject/govobject.md)    
```

Reply:
Nice catch!